autonomy/constants.py

```diff
@@ -56,6 +56,6 @@
     f"{SERVICE_REGISTRY_IMAGE_NAME}:{SERVICE_REGISTRY_IMAGE_VERSION}"
 )
 ACN_IMAGE_NAME = os.environ.get("ACN_IMAGE_NAME", "valory/open-acn-node")
-
-OAR_IMAGE = "valory/oar-{agent}:{version}"
+DEFAULT_DOCKER_IMAGE_AUTHOR = "valory"
```

Comment:
I think the default author should be set using `aea.cli.utils.config.get_default_author_from_cli_config`

Reply:
@angrybayblade 
dont think it's a good idea to perform on module load operations with config reading.
it will be done every time module loaded even no need in this variable.

also not sure image author and agent author are the same.

@8ball030  any ideas?

Reply:
I agree, but I think it'll be really confusing if we have multiple types of authors. I propose we don't use the `DEFAULT_DOCKER_IMAGE_AUTHOR` variable for default author, we pass down `None` if the author name is not provided on the command and use `get_default_author_from_cli_config` method where the author name is required

Reply:
@angrybayblade updated